uk.co.compucorp.civicrm.hrleaveandabsences/templates/CRM/HRLeaveAndAbsences/Form/ManageEntitlements.tpl

```diff
@@ -98,7 +98,7 @@
   </tbody>
 </table>
 <div class="crm-submit-buttons">
-  <a href="javascript:history.back()" class="button"><span>{ts}Back{/ts}</span></a>
+  <a href="{$smarty.get.returnUrl}" class="button"><span>{ts}Back{/ts}</span></a>
```

Comment:
Ideally, it's a bad practice for templates to access the global scope. Any value required by the template must be passed in to it by the controller. In this case, the controller is `uk.co.compucorp.civicrm.hrleaveandabsences/CRM/HRLeaveAndAbsences/Form/ManageEntitlements.php`. To pass a variable to a template, you can use the `assign` method, inside the `buildQuickForm` method of the controller.